charts/rollups-node/templates/database-secret.yaml

```diff
@@ -4,6 +4,7 @@ SPDX-License-Identifier: Apache-2.0 (see LICENSE)
 */}}
 
 ---
+{{- if ne .Values.postgresql.endpoint "secret" }}
```

Comment:
I don't get it.

We're using the helper function `validator.postgresEndpoint` in this secret, so that we define the `POSTGRES_ENDPOINT` data value for the secret according to what was defined (or not) in the values file.

IMO, we should allow the chart user to define which secret to use. Like we do with the `.Values.mnemonic.secretRef`

Reply:
Here's my idea : 

use `postgresql.endpoint` as a central config. if we define the value to `secret`, then this secret do not need to create and the chart will use the existing secret (like we can create it with external-secret). but if this value is set to anything except the `secret`. this k8s secret will create.

Reply:
Reference : https://github.com/cartesi/cloud-infra/pull/79/commits/9d1c643e1309a094530061b78043ee226d655326

Reply:
First, defining `.endpoint` value to `secret` is not go, since `secret` is not a valid endpoint, it will lead to confusion.

And even if we used this approach, how to know which secret will be used?

Reply:
we already have a template for secret name : `{{ "validator.fullname" . }}-database`

So, I'm using this to create secret with external secret in my PR in cloud-infra.

Reply:
Don't mix our deployment details with the chart definition.

A user can have single or multiple databases, and they may want to define a different secret for each node deployment with arbitrary secret names.

We can define the name for our secret when use use definitions from the helper method, but if we want to provide the user the ability to point to another secret, it has to be any secret name they want. The same way we do with `.values.mnemonic.SecretRef`, as I said in my first comment.

Reply:
So, I can add a field like `.Values.postgresql.SecretRef`  and change the helm chart behaviour when it's defined.